docs/spec/lsm/adr-002.md

```diff
@@ -0,0 +1,362 @@
+
+
+A user would be able to visit any liquid staking provider that has integrated with the LSM and click a button to convert his staked tokens to liquid staked tokens. It would be as easy as liquid staking unstaked tokens.
+
+Technically speaking, this is accomplished by using something called an “LSM share.” Using the liquid staking module, a user can tokenize their staked tokens and turn it into LSM shares. LSM shares can be redeemed for underlying staked tokens and are transferable. After staked tokens are tokenized they can be immediately transferred to a liquid staking provider in exchange for liquid staking tokens - without having to wait for the unbonding period.
+
+
+## Economics:
+
+We expect that eventually governance may decide that the principal agent problems between validators and liquid staking are resolved through the existence of mature liquid staking synthetic asset systems and their associate risk framework. Governance can effectively disable the feature by setting the scalar value to -1 and allow unlimited minting and all liquid delegations to be freely undelegated.
+
+During the transitionary period, this creates a market for liquid shares that may serve to help further decentralize the validator set.
+
+It also allows multiple participants in a validator business to hold their personal stakes in segregated accounts but all collectively contribute towards demonstrating alignment with the safety of the protocol.
+
+
+## Technical Spec
+
+### Modification from ADR-001 
+ADR-002 introduces the global liquid staking cap, validator liquid staking cap, and renames exemption factor to validator bond factor. Delegations from 32-length addresses and LSM tokenized shares are tracked against the global liquid staking, validator liquid staking cap, and validator bond caps. This requires changing the standard staking transactions to introspect. The reason for checking the account type is because ICAs and tokenize share record module accounts have 32-length addresses, so in practice this limits liquid staking. To be clear, any ICA or module account staking is counted against this cap - not just ICA delegations from liquid staking providers.
+
+### Instructions for validators
+Once delegated to a validator, a delegator or validator can convert their delegation to a validator into Validator Bond by signing a ValidatorBond message. 
+
+The ValidatorBond message is exposed by the staking module and can be executed as follows:
+```
+gaiad tx staking validator-bond cosmosvaloper13h5xdxhsdaugwdrkusf8lkgu406h8t62jkqv3h <delegator> --from mykey  
+```
+There are no partial Validator Bonds: when a delegator or validator converts their shares to a particular validator into Validator Bond, their entire delegation to that validator is converted to Validator Bond. If a validator or delegator wishes to convert only some of their delegation to Validator Bond, they should transfer those funds to a separate address and Validator Bond from that address.
```

Comment:
ditto

Reply:
Can a delegator redelegate to the same validator? Basically, if delegator D has 100 atoms bonded with validator V1 and wants to ValidatorBond only 70 with V1 (without unbonding any of the 100 atoms), it must redelegate 30 to another validator V2, right?